app/javascript/src/Users/components/FeatureAccessInput.jsx

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import * as React from 'react'
+
+import { getFeatureName, isServicePermissionsGranted } from 'Users/utils'
+
+import type { Feature, AdminSections } from 'Users/types'
+
+type Props = {
+  features: Feature[],
+  checkedAdminSections: AdminSections[],
+  showServices?: boolean,
+  onChange: AdminSections => void
+}
+
+const FeatureAccessInput = ({ features, checkedAdminSections, showServices = false, onChange }: Props) => {
+  const olClass = `FeatureAccessList ${showServices ? '' : 'FeatureAccessList--noServicePermissionsGranted'}`
```

Comment:
`FeatureAccessListClass`?

Reply:
I see you use it in other places the same way, denoting just the html structure related class. It's only good when there's only one element of each, either way, better to use specific names.